README.md

```diff
@@ -299,7 +299,76 @@ The default XML parser contained a security risk which made the driver prone to
 
 | Parameter       | Value           | Required      | Description  |
 | ------------- |:-------------:|:-------------:| ----- |
-|`allowXmlUnsafeExternalEntity` | Boolean | No | Set to true if you would like to use XML inputs that refer to external entities. WARNING: Setting this to true is unsafe since your system to be prone to XXE attacks.<br/><br/>**Default value:** `false` | 
+|`allowXmlUnsafeExternalEntity` | Boolean | No | Set to true if you would like to use XML inputs that refer to external entities. WARNING: Setting this to true is unsafe since your system to be prone to XXE attacks.<br/><br/>**Default value:** `false` |
+
+### AWS IAM Authentication
+
+An optional authentication method is to use Amazon AWS Identity and Access Management (IAM). 
+When using AWS IAM Authentication, Connection URL must be an Amazon endpoints, and not a custom domain or IP Address. 
+
+IAM database authentication is available for the following database engines:
```

Comment:
Can we be consistent with our wording? Either `AWS IAM authentication` or `IAM database authentication`.
Instead of putting the availability here, can we provide a link to the AWS documentation? The documentation also talks about limitations and recommendation. 

https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/UsingWithRDS.IAMDBAuth.html#UsingWithRDS.IAMDBAuth.Limitations